rules/windows/credential_access_forced_authentication_pipes.toml

```diff
@@ -0,0 +1,74 @@
+[metadata]
+creation_date = "2024/07/23"
+integration = ["endpoint", "system"]
+maturity = "production"
+updated_date = "2024/07/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a potential forced authentication using related SMB named pipes. Attackers may attempt to force targets to
+authenticate to a host controlled by them to capture hashes or enable relay attacks.
+"""
+from = "now-9m"
+index = ["logs-system.security-*", "logs-windows.forwarded-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Potential Forced Authentication - SMB Named Pipes"
+references = [
+    "https://github.com/p0dalirius/windows-coerced-authentication-methods",
+    "https://www.thehacker.recipes/a-d/movement/mitm-and-coerced-authentications",
+    "https://attack.mitre.org/techniques/T1187/",
+]
+risk_score = 21
+rule_id = "73d05fd0-d73b-4625-aa2e-3221965aa2e3"
+setup = """## Setup
+
+The 'Audit Detailed File Share' audit policy must be configured (Success Failure).
+Steps to implement the logging policy with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Policies >
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+Audit Policies >
+Object Access >
+Audit Detailed File Share (Success,Failure)
+```
+"""
+severity = "low"
+tags = [
+    "Domain: Endpoint",
+    "OS: Windows",
+    "Use Case: Threat Detection",
+    "Tactic: Credential Access",
+    "Data Source: Elastic Defend",
+    "Data Source: Active Directory",
+    "Use Case: Active Directory Monitoring",
+]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+sequence by source.ip, user.name with maxspan=1s
+[authentication where host.os.type == "windows" and event.action == "logged-in" and
+ winlog.event_data.AuthenticationPackageName : "NTLM" and winlog.event_data.SubjectUserSid == "S-1-0-0"]
+[file where host.os.type == "windows" and event.code == "5145" and file.name : ("Spoolss", "netdfs", "lsarpc", "lsass", "netlogon", "samr", "efsrpc", "FssagentRpc")]
```

Comment:
Had some unrelated hits in the `lsarpc` pipe, will measure the noise as this one is used in EFSRPC related forced auth, and will remove it and create a BBR if the volume is too high

Reply:
Thanks! I'll work on that